### PR TITLE
add learning-sessions document

### LINKED ISSUE #34: Add learning sessions document
As discussed in the Nov 15 SIG call, we want to add a document to capture any sessions that describe various security related tools similar to https://github.com/kubernetes/sig-security/blob/main/sig-security-tooling/learning-sessions.md

### PR DESCRIPTION
**Pull request by codeboten**
Fixes #34

### CODE REVIEW

File: docs/learning-sessions.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##  Learning Sessions
+
+- For the most up to date information, please visit the [Sig Security Meeting Notes](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg).
+- Also, please check out the slack channel at [#otel-sig-security](https://cloud-native.slack.com/archives/C05A85QC281).
+
+Here is the list of learning sessions hosted by sig-security tooling community:
+
+| **Date**   | **Topic**                                                              | **Speaker(s)**                                                                                          | **Link**                                     |
+| ---------- | :--------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------------- | :------------------------------------------: |
+| 03-29-2023 | [cosign](https://github.com/sigstore/cosign)                           | [Carlos Tadeu Panato Junior](https://github.com/cpanato)                                                |   |
```
Comment (codeboten) [inline docs/learning-sessions.md:10]:
```suggestion
| TBD        | [cosign](https://github.com/sigstore/cosign)                           | [Carlos Tadeu Panato Junior](https://github.com/cpanato)                                                |   |
```

**Comment by codeboten** [comment]
@jpkrohling @cartersocha here's the doc I suggested at the meeting yesterday

File: docs/learning-sessions.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+##  Learning Sessions
+
+- For the most up to date information, please visit the [Sig Security Meeting Notes](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg).
+- Also, please check out the slack channel at [#otel-sig-security](https://cloud-native.slack.com/archives/C05A85QC281).
+
+Here is the list of learning sessions hosted by the sig-security community:
+
+| **Date**   | **Topic**                                                              | **Speaker(s)**                                                                                          | **Link**                                     |
+| ---------- | :--------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------------- | :------------------------------------------: |
+| TBD        | [cosign](https://github.com/sigstore/cosign)                           | [Carlos Tadeu Panato Junior](https://github.com/cpanato)                                                |                                              |
```
Comment (jpkrohling) [inline docs/learning-sessions.md:10]: @cpanato, do you remember when was the presentation you've given to the Collector SIG? Would you be open to doing another one for the SIG Security? We'd then record and make it available for other folks in the community.

Comment (cpanato) [reply docs/learning-sessions.md:10]: I don't remember exactly what i spoke but we can redo that :)

Comment (codeboten) [reply docs/learning-sessions.md:10]: Thanks @cpanato! would be great to have you at next Wednesday's SIG call (29-Nov) if that works for you!

Comment (cpanato) [reply docs/learning-sessions.md:10]: @codeboten sorry for this week and next I cannot join :(, but I can plan to join in two weeks

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**